src/content/docs/tutorial-create-alerts/create-chart-alerts.mdx

```diff
@@ -0,0 +1,121 @@
+
+3. If you want to add another threshold, select either **Add threshold** or **Add lost signal threshold**, depending on your needs. Lost signal thresholds are important for monitoring entities that may stop reporting or queries that may [return null values](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#query-order).
+
+4. Select **Next**
+    </Step>
+    <Step>
+## Add alert condition details
+
+<img
+    title="Alert condition details"
+    alt="A screenshot displaying how to create the details for an alert"
+    src={alertsDetails}
+/>
+
+1. Add a name for your alert in the **Name** field. This will help you identify this condition, as well as find the condition in the future in case you need to edit or remove it.
+
+2. Choose the policy for this condition to connect to. If you have an **Existing policy**, select it from the drop-down. If not, you'll need to [create a policy](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/#alert-policy-name) by selecting **New policy** and creating it from here.
+
+3. Set any of the additional settings for the alert. This includes setting how long until open incidents are automatically closed and entering any [custom descriptions](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/alert-custom-violation-descriptions/) for the incidents.
+
+4. Select **Save condition** to finish.
+    </Step>
+</Steps>
+
+You've now created an alert condition, and helped to make identifying symptoms within your system that much easier! While there are many options to select when creating an alert condition, the basic process when starting from a chart will always follow the steps above. 
+
+If that was your first alert condition, feel free to create more from your charts with different options to get used to how they work. You can also create conditions from NRQL queries, which is the next topic in this tutorial series.
+
+## Get more from alerting
+
+We have some advanced alerting features that can help you make alerting more efficent and manageable for your team.
+
+* If you want greater control over where and when you receive alerts and ensure the right people are notified about issues, see how to use [Workflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/).
+* If you want to enable near-instant notification of critical alerts, helping you reduce your mean time to resolution (MTTR), see how to use our [anomaly detection with applied intelligence](/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence/).
+* If you want to use APIs to manage New Relic alerts, we have [two options](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/alerts-api/intro-alerts-api/) you can use to do so.
```

Comment:
`two options` sounds like a PITA, but i think if you name the options it sounds enabling:

> you can do so with either REST API or GraphQL.